src/tools/scripts/resetInvalidDateDeNaissance/resetInvalidDateDeNaissance.js

```diff
@@ -0,0 +1,21 @@
+const { execute } = require('../../utils');
+
+execute(__filename, async ({ db, logger, Sentry, exit }) => {
+  try {
+    await db.collection('conseillers').updateMany({
+      $or: [
+        { dateDeNaissance: { $lte: new Date('1920-01-01T00:00:00.000Z') } },
+        { dateDeNaissance: { $gte: new Date('2000-01-01T00:00:00.000Z') } }
+      ]
+    }, {
+      $unset: { dateDeNaissance: true, sexe: true }
```

Comment:
yes avec le $unset, il me semble que true n'était pas nécessaire, juste des quotes 
```suggestion
      $unset: { dateDeNaissance: '', sexe: ''}
```

Reply:
Fait